data_folder/config.yaml

```diff
@@ -53,3 +53,5 @@ job_applicants_threshold:
 llm_model_type: openai
 llm_model: 'gpt-4o-mini'
 # llm_api_url: 'https://api.pawan.krd/cosmosrp/v1'
+
+browser: firefox
```

Comment:
move this to config.py

Reply:
updated, moved to `config.py` as the variable `BROWSER_TYPE_CONFIG`